### PR TITLE
UI Bugfix:  Remove black empty bar in tx-table after searching  with no results

### PR DESCRIPTION
**Pull request by relativisticelectron**
In https://github.com/cryptoadvance/specter-desktop/issues/1834 there was a blue bar appearing in tx-table 
image

This PR fixes that.

EDIT:  The visible part is a black empty bar.  But if I understand correctly, it is caused by the small blue-background in the beginning of the empty row.

### CODE REVIEW

File: src/cryptoadvance/specter/templates/includes/tx-table.html [hunk 1]
```diff
@@ -1005,6 +1005,7 @@ <h1>{{ _("Export transactions to CSV") }}</h1>
                     this.pageCount = 1;
                 } else {
                     this.tableHeader.classList.remove('hidden');
+                    this.feedbackRow.classList.remove('feedback-row');
```
Comment (moneymanolis) [inline src/cryptoadvance/specter/templates/includes/tx-table.html:1008]: Not sure how this relates to the blue bar, also not necessary for the feedbackRow, we are adding `hidden` here already.

**Comment by moneymanolis** [comment]
Not sure, I am following, if you want to get rid of the focus blue highlight, you'd need that:
```
        .search:focus {
            outline: none;
        }
```
See also here: https://stackoverflow.com/questions/7474147/input-field-blue-focus-highlight-impossible-to-get-rid-of

Let me know whether you wanted that and I can push that.

File: src/cryptoadvance/specter/templates/includes/tx-table.html (hunk 1 above)
Comment (relativisticelectron) [reply src/cryptoadvance/specter/templates/includes/tx-table.html:1008]: Well, I noticed that the page originally starts from `class="hidden"` and then later `class='feedback-row'` is added and not removed again. It does cause the blue bar even though "hidden" is added.

**Comment by relativisticelectron** [comment]
I think the blue surrounding bar is OK, and consistent with other input fields.

**Comment by moneymanolis** [comment]
Wait, I think I understand now, you not referring to the blue focus outline but to this:

grafik

**Comment by moneymanolis** [comment]
During testing I came across this, will create an issue out of it. Since there are so many issues with the search bar, I will create a milestone for this, so we can gather all related issues with the milestone tag.

The feedback in the tx table search is misleading / incorrect when there are no hits. The feedback shown is intended for a fresh wallet or a wallet without rescan.

grafik

**Pull request merged**